packages/components/README.md

```diff
@@ -35,7 +35,7 @@ _If you're using [`Popover`](/packages/components/src/popover/README.md) or [`To
 
 By default, the `Popover` component will render within an extra element appended to the body of the document.
 
-If you want to precisely contol where the popovers render, you will need to use the `Popover.Slot` component.
+If you want to precisely control where the popovers render, you will need to use the `Popover.Slot` component.
```

Comment:
This is definitely a typo, but since it is outside the scope of this PR and is a change related to a specific package, it would be better to submit it as a separate PR.

Reply:
Absolute URL is applied. Fix a collection of typos PR is coming ;)